great_ape_safe/ape_api/badger.py

```diff
@@ -175,7 +175,7 @@ def claim_bribes_hidden_hands(self, claim_from_strat=True) -> dict:
         aggregate = {"tokens": [], "amounts": []}
         for item in data:
             aggregate["tokens"].append(item["token"])
-            aggregate["amounts"].append(item["claimMetadata"]["amount"])
+            aggregate["amounts"].append(item["claimable"].replace('.', ''))
 
         metadata = [(item['claimMetadata']['identifier'], item['claimMetadata']
                      ['account'], item['claimMetadata']['amount'], item['claimMetadata']['merkleProof'])
```

Comment:
Shouldn't we also change it here? Not sure what the purpose of `amount` is then...

Reply:
i think the amount sent as metadata is more related to the proof and the merkle tree, not the actual individual claimable.